backend/initialRecommendations.js

```diff
@@ -0,0 +1,230 @@
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+// Fetch liked interactions for a user
+async function fetchLikedInteractions(userId) {
+  const interactions = await prisma.interaction.findMany({
+    where: {
+      userId: userId,
+      liked: true,
+    },
+    include: {
+      Business: true,
+      User: true,
+    },
+  });
+  return interactions;
+}
+
+// Analyze user preferences from liked interactions
+function analyzeUserPreferences(interactions) {
+  const totalLikes = interactions.length;
```

Comment:
This variable looks like it should be called something else, since it's looking a the total number of interactions, not just likes.

Reply:
It is actually looking for the total number of "liked" businesses only, not all interactions. But you're right about that.